tests/Logger/TextFileLoggerTest.php

```diff
@@ -276,7 +277,12 @@ public function test_it_logs_correctly_with_mutations_and_debug_verbosity_and_no
 
 Diff Diff
 
-TXT
+TXT;
+
+        $content = str_replace("\n", PHP_EOL, $content);
```

Comment:
I seems like other places about would also fancy this approach with `str_replace`. It is way more readable than `'foo'.PHP_EOL.'bar'`.